### PR TITLE
Install Ansible Galaxy dependencies in setup

### PR DESCRIPTION
**Pull request by nre-ableton**
Should have been done in af81c21.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -7,8 +7,11 @@ devToolsProject.run(
   defaultBranch: 'main',
   setup: { data ->
     Object venv = pyenv.createVirtualEnv(readFile('.python-version'))
-    venv.run('pip install -r requirements-dev.txt')
-    data['venv'] = venv
+    venv.inside {
+      sh 'pip install -r requirements-dev.txt'
+      ansibleUtils.galaxyInstall()
+    }
+    ansibleUtils.galaxyInstall()
```
Comment (rco-ableton) [inline Jenkinsfile:14]: Both inside _and_ outside the venv?

Comment (nre-ableton) [reply Jenkinsfile]: Ah damn, deleted the wrong line. See 09d12f7.

**Review by rco-ableton — APPROVED** [review]

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Comment by nre-ableton** [comment]
@ablbot runci

**Pull request merged**